test/automation/src/positron/positronConsole.ts

```diff
@@ -73,7 +79,8 @@ export class PositronConsole {
 	): Promise<InterpreterInfo | undefined> {
 		const interpreterElem = await this.selectInterpreter(
 			desiredInterpreterType,
-			desiredInterpreter
+			desiredInterpreter,
+			true
```

Comment:
I'm surprised there are so many cases where we want to skip waiting for the console to be ready 😮 In particular, I think the project wizard tests that use this code path needed to wait for the interpreter due to failures we saw on @jonvanausdeln's Windows machine without waiting, so I'd be curious if those tests still pass.

Reply:
Well, if these were "startup tests", then yes, they would probably need to wait, but these tests are performed on an instance of Positron that has already executed some tests so we can assume the system is stable and we are not awaiting the startup of any default interpreter.

Reply:
@jonvanausdeln can you run the following tests on Windows as a cross check:
* project wizard
* editor
* variables
* shiny
* console clipboard
* console history
?

Reply:
Perhaps selectAndGetInterpreter should also take a "skipWait" param for future use?  Here we cannot wait else the test will fail but perhaps in the future a usage will come up when it does make sense.